score.py

```diff
@@ -229,8 +229,9 @@ def convert_nifti_to_zipped_dicom_rt(nifti_file: Path, reference_series: Path, s
         modelId=model_id
     )
     # Log stdout, stderr from DICOM-RT conversion.
-    logging.debug("stdout: %s", stdout)
-    logging.debug("stderr: %s", stderr)
+    logging.info("DICOM-RT conversion stdout: %s", stdout)
+    if stderr != "":
+        logging.error("Errors detected during DICOM conversion: %s", stderr)
```

Comment:
Should we simply fail here if there's something in stderr or a non-zero exit code? Otherwise it will raise a filenotfound exception that is completely nondescript.

Reply:
Good point!